src/cart/crt_swim.h

```diff
@@ -15,8 +15,8 @@
 #include "swim/swim_internal.h"
 
 #define CRT_SWIM_FLUSH_ATTEMPTS		100
-#define CRT_SWIM_PROGRESS_TIMEOUT	0	/* minimal progressing time */
-#define CRT_DEFAULT_PROGRESS_CTX_IDX	0
+#define CRT_SWIM_PROGRESS_TIMEOUT	300	/* minimal progressing time in msec */
+#define CRT_DEFAULT_PROGRESS_CTX_IDX	2
```

Comment:
sorry, I did not get why SWIM CTX_IDX is always 2?
RDB XS (XS0) create cart context idx 0, and then create SWIM cart context right? not sure why the IDX is 2?
I may misunderstood sth.

Reply:
yes, you right. SYS XS: 0, SWIM XS: 1, DRPC XS: 2.